src/core/core.scale.js

```diff
@@ -435,16 +445,17 @@ module.exports = function(Chart) {
 					// TODO - improve this calculation
 					var labelHeight = (sinRotation * largestTextWidth)
 						+ (tickFont.size * tallestLabelHeightInLines)
-						+ (lineSpace * tallestLabelHeightInLines);
+						+ (lineSpace * (tallestLabelHeightInLines - 1))
```

Comment:
What's the reason for this change? It looks equivalent to the old code

Reply:
It is actually equivalent but prepares for implementing tick label padding. `lineSpace` is only needed lines-1 times. The last `lineSpace` is really outer padding and could indeed be any other value.

Reply:
Gotcha